sdks/go/pkg/beam/core/funcx/fn.go

```diff
@@ -243,6 +247,16 @@ func (u *Fn) Window() (pos int, exists bool) {
 	return -1, false
 }
 
+// Pane returns (index, true) iff the function expects a PaneInfo.
+func (u *Fn) Pane() (pos int, exists bool) {
```

Comment:
I won't ask you to backfill tests for all the other similar methods, but can we ad a tests for this method, for the function having a Pane and not having a pane?

Reply:
got it.